app/src/main/java/com/teamltt/carcare/service/ObdBluetoothService.java

```diff
@@ -368,6 +369,7 @@ protected Void doInBackground(Void... ignore) {
                     commands.add(RuntimeCommand.class);
                     commands.add(SpeedCommand.class);
                     commands.add(RPMCommand.class);
+                    commands.add(FuelEconomyCommand.class);
```

Comment:
I will merge this if you comment out or take out the FuelEconomyCommand class. The rest of the stuff is good, but this line is simply untestable.